src/renderer/components/menu/menu-actions.tsx

```diff
@@ -125,32 +124,31 @@ export class MenuActions extends React.Component<MenuActionsProps> {
     return (
       <>
         {this.renderTriggerIcon()}
-        <RenderDelay>
```

Comment:
@aleksfront Wasn't this rendering delay added on purposes to help with initial load times?

Reply:
We discussed with @aleksfront about the requirement of `RenderDelay` and came to decision that it's not needed. We didn't saw any impact on initial load times.